tests/vcd_user.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import string
+import random
+
+from pyvcloud.vcd.org import Org
+from pyvcloud.vcd.test import TestCase
+
+
+class TestUser(TestCase):
+    def test_create_user(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, is_admin=True, resource=logged_in_org)
```

Comment:
see previous comment about is_admin

Reply:
It depends on the org value in config.yml file. is_admin here is unnecessary. So I am removing it.